src/Sylius/Bundle/PayumBundle/Validator/Constraints/GatewayConfig.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\PayumBundle\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+final class GatewayConfig extends Constraint
```

Comment:
```suggestion
final class GatewayConfiguration extends Constraint
```
or
`GatewayConfigurationValidator` to `GatewayConfigValidator`